forms-flow-web/src/components/ServiceFlow/details/TaskHeaderListView.js

```diff
@@ -53,20 +53,28 @@ const TaskHeaderListView = React.memo(({task,taskId,groupView = true}) => {
   }, [task?.due]);
 
   const updateBpmTasksAndDetails = (err) =>{
-    if (!err) {
-      if (!SocketIOService.isConnected()) {
-        if (selectedFilter) {
-          dispatch(getBPMTaskDetail(taskId));
-          dispatch(
-            fetchServiceTaskList(reqData,null,firstResult)
-          );
-        } else {
-          dispatch(setBPMTaskDetailUpdating(false));
-        }
-      }
-    } else {
+    // if (!err) {
+    //   if (!SocketIOService.isConnected()) {
+    //     if (selectedFilter) {
+    //       dispatch(getBPMTaskDetail(taskId));
+    //       dispatch(
+    //         fetchServiceTaskList(reqData,null,firstResult)
+    //       );
+    //     } else {
+    //       dispatch(setBPMTaskDetailUpdating(false));
+    //     }
+    //   }
+    // } else {
+    //   dispatch(setBPMTaskDetailUpdating(false));
+    // }
+    // Above code commented and added below 3 lines for refreshing the tasks on each update operation without checking conditions.
+    if(err)
```

Comment:
this code will work only in err scenario right so what about if no error is there  @arun-s-aot

Reply:
sonarcloud

Reply:
only console log need to be printed if there is error. rest of the lines should run in both error and success cases @shuhaib-aot .

will check sonar issue